packages/office-ui-fabric-react/tslint.json

```diff
@@ -14,7 +14,7 @@
     "no-any": false,
     "variable-name": false,
     "no-internal-module": false,
-    "prefer-const": false,
+    "prefer-const": true,
```

Comment:
This should just be deleted, and you should update the default oufr-tslint/tslint.json to be true.

Reply:
Got it. Let me fix that.

Reply:
done

---
In reply to: [168983208](https://github.com/OfficeDev/office-ui-fabric-react/pull/4018#discussion_r168983208) [](ancestors = 168983208)